.kres.yaml

```diff
@@ -33,15 +35,22 @@ spec:
     - liburcu
     - linux-firmware
     - lvm2
+    - mtools
     - musl
     - openssl
     - pcre2
+    - pigz
```

Comment:
Are all these absolutely necessary? Maybe after this is tested it'd be a great idea to check and replace some with Go alternatives

Reply:
yeh, replacing with go native is planned for 1.11